examples/java/spring-boot/src/main/java/uk/gov/api/springboot/services/MetadataService.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.api.springboot.services;
+
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+import uk.gov.api.springboot.daos.MetadataDao;
+import uk.gov.api.springboot.models.metadata.ApiMetadata;
+import uk.gov.api.springboot.models.metadata.Data;
+import uk.gov.api.springboot.repositories.MetadataRepository;
+
+@Service
+public class MetadataService {
+
+  private MetadataRepository repository;
+
+  public MetadataService(MetadataRepository repository) {
+    this.repository = repository;
+  }
+
+  public List<ApiMetadata> retrieveAll() {
+    return repository.findAll().stream().map(convertToApiMetadata()).collect(Collectors.toList());
+  }
+
+  private Function<MetadataDao, ApiMetadata> convertToApiMetadata() {
```

Comment:
This could be static:
```suggestion
  private static Function<MetadataDao, ApiMetadata> convertToApiMetadata() {
```